index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portfolio Site</title>
+  <link rel="stylesheet" href="style.css">
+  <link rel="stylesheet" href="//use.fontawesome.com/releases/v5.0.7/css/all.css">
+  
+</head>
+<body>
+  <header class="head">
+    <nav>
+      <h3 class="logo">Portfolio</h3>
```

Comment:
- [ ] Don't forget the logo must have a link when clicked. Kindly change `h3 tag` to `a tag`. :smile_cat: